### PR TITLE
Add admin redirect, fix card errors not being caught

### LINKED ISSUE #101: Make /admin URL work without trailing slash
/admin/ is the admin, but /admin 404s.

### PR DESCRIPTION
**Pull request by dracos**
Fixes https://github.com/mysociety/mapit.mysociety.org/issues/101
Hopefully fixes issue with card declining at point customer is created.

### CODE REVIEW

File: subscriptions/views.py [hunk 1]
```diff
@@ -217,7 +217,18 @@ def _add_subscription(self, form_data):
         if form_data['payment_method']:
             cust_params['payment_method'] = form_data['payment_method']
             cust_params['invoice_settings'] = {'default_payment_method': form_data['payment_method']}
-        obj = stripe.Customer.create(**cust_params)
+
+        # At the point the customer is created, details such as postcode and
+        # ssecurity code can be checked, and therefore fail
```
Comment (davea) [inline subscriptions/views.py]: Small typo on `ssecurity`.

**Review by davea — APPROVED** [review]
Looks good, might be worth adding a test for the Stripe user creation error if that's simple enough to do?

File: subscriptions/views.py (hunk 1 above)
Comment (dracos) [reply subscriptions/views.py]: 🐍

**Comment by dracos** [comment]
Added test was easy enough once I worked out how mocks worked again :)

**Pull request merged**